dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/MessageTraceFileNameTimedRolloverTest.java

```diff
@@ -0,0 +1,273 @@
+        server.setMarkToEndOfLog();
+        server.setServerConfigurationFile("server_trace_log.xml");
+        assertNotNull("Configuration update should complete",
+                      server.waitForConfigUpdateInLogUsingMark(null));
+
+        // Verify trace.log now exists after dynamic update
+        boolean traceLogNowExists = server.fileExistsInLibertyServerRoot("logs/trace.log");
+        assertTrue("REQUIREMENT: trace.log should exist after dynamic update to traceFileName=trace.log", traceLogNowExists);
+    }
+
+    /**
+     * Test 4: Configure TimedLogRollover (in server.env), and then dynamically update the traceFileName
+     * to test.log in server.xml, verify if the new test.log gets rolled over,
+     * and the trace.log file does NOT roll over.
+     */
+    @Test
+    public void testDynamicTraceFileNameToCustomFile() throws Exception {
+        // Start with default configuration
+        server.startServer();
+        server.waitForStringInLog("CWWKF0011I");
+
+        // Dynamically update server.xml to change traceFileName to test.log
+        server.setMarkToEndOfLog();
+        server.setServerConfigurationFile("server_test_log.xml");
+
+        // Wait for config update
+        assertNotNull("Configuration update should complete",
+                      server.waitForConfigUpdateInLogUsingMark(null));
+
+        // Verify new test.log gets created
+        boolean testLogNowExists = server.fileExistsInLibertyServerRoot("logs/test.log");
+        assertTrue("REQUIREMENT: test.log should exist after dynamic update to traceFileName=test.log", testLogNowExists);
+    }
+
+    /**
+     * Test 5: Repeat scenario 4 using messageFileName, instead.
+     * Configure TimedLogRollover (in server.env), then dynamically update messageFileName
+     * to custom file, verify rollover behavior.
+     */
+    @Test
+    public void testDynamicMessageFileNameToCustomFile() throws Exception {
+        // Start with default configuration
+        server.startServer();
+        server.waitForStringInLog("CWWKF0011I");
+
+        // Dynamically update server.xml to change messageFileName to custom_message.log
+        server.setMarkToEndOfLog();
+        server.setServerConfigurationFile("server_custom_message.xml");
+
+        // Wait for config update
+        server.waitForStringInLogUsingMark("CWWKG0017I");
+
+        // Verify new custom_message.log gets created
+        boolean customMessageNowExists = server.fileExistsInLibertyServerRoot("logs/custom_message.log");
+        assertTrue("REQUIREMENT: custom_message.log should exist after dynamic update to messageFileName=custom_message.log", customMessageNowExists);
+    }
+
+    /**
+     * Test 6: Configure traceFileName=stdout in server.xml, verify the trace.log is NOT created.
+     * (This scenario depends on issue #31949 - Cannot stop generating trace.log file)
+     */
+    @Test
+    public void testTraceFileNameStdoutNoTraceLogCreated() throws Exception {
```

Comment:
Better to disable this test, until the the following [PR](https://github.com/OpenLiberty/open-liberty/pull/32215/files) is delivered